### PR TITLE
Fix for CTRL+ALT key combinations ...

### PR DESCRIPTION
**Pull request by entrhopi**
and prevent some actions to trigger multiple when key is held down

### CODE REVIEW

**Review by greaka — APPROVED** [review]

**Review by agaertner — APPROVED** [review]

File: Blish HUD/Controls/TextInputBase.cs [hunk 1]
```diff
@@ -187,8 +187,12 @@ public abstract class TextInputBase : Control {
         /// </summary>
         public int Length => _text.Length;
 
+        /// <summary>
+        /// Get state of modifier keys
+        /// </summary>
```
Comment (dlamkins) [inline Blish HUD/Controls/TextInputBase.cs]: Just FYI, this will only apply this summary to the `IsShiftDown`.  It won't apply it to the two below it.

**Review by dlamkins — CHANGES_REQUESTED** [review]
Changes look great!  Minor thing about the XML doc tag if you could either remove it or add one for each of the three calls there.  

If you just meant it as a more typical comment, we could probably just remove the XML portion of the tags and leave:

```cs
// Get state of modifier keys
```

**Review by dlamkins — APPROVED** [review]

**Comment by itsShoww** [comment]
I'm very new to github and Blishhud too. I wasn't sure how to communicate with the people developing these modules (this is the only way I found) but I came across a post and a GIF about the Special Forces module (rotation addon) and I was wondering if it's already working/released, I've downloaded the "Blish hud modules master" zip folder and placed it on the modules folder but whenever I launch the game and blish, it says "you currently dont have any modules". I'd love to try this out, any help is appreciated, and sorry if this is totally the wrong way to contact you guys.

**Comment by greaka** [comment]
Hey there!
This is indeed not the optimal place to ask such a question.
If you have trouble installing Blish-HUD or modules, visit https://blishhud.com/ for a guide.
If you still have questions after that, come and chat with us on our Discord server!
https://discord.gg/zhj23Sv5mA
If you don't have or want to use Discord, then open a new issue on this page please. You find issues at the top of this site.

**Pull request merged**